mne/preprocessing/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+
+# License: BSD (3-clause)
+
+import numpy as np
+
+from ..utils import logger
+
+
+# adapted from SciPy
+
+def check_COLA(win, nperseg, noverlap, tol=1e-10):
+    """Check whether the Constant OverLap Add (COLA) constraint is met."""
+    nperseg = int(nperseg)
+    if nperseg < 1:
+        raise ValueError('nperseg must be a positive integer')
+    if noverlap >= nperseg:
+        raise ValueError('noverlap must be less than nperseg.')
+    noverlap = int(noverlap)
+    step = nperseg - noverlap
+    binsums = np.sum((win[ii * step:(ii + 1) * step]
+                      for ii in range(nperseg // step)), axis=0)
+    if nperseg % step != 0:
+        binsums[:nperseg % step] += win[-(nperseg % step):]
+    deviation = binsums - np.median(binsums)
+    return np.max(np.abs(deviation)) < tol
+
+
+def _get_lims_COLA(n_samp, n_times, sfreq, picks):
```

Comment:
what is this _get_lims_COLA ?

also it should be _get_lims_cola

we don't put capitals in function names.

Reply:
acronyms should be capitalized, it's the "Constant OverLap Add" constraint